src/main/java/jp/ac/titech/c/se/stein/core/RepositoryRewriter.java

```diff
@@ -79,6 +69,18 @@
     @Option(names = "--extra-attributes", description = "rewrite encoding and signature in commits", order = Config.MIDDLE)
     protected boolean isRewritingExtraAttributes = false;
 
+    enum CacheLevel {
+        Blob, Tree, Commit
+    }
+
+    @Option(names = "--cache-level", arity = "0..*", description = {
+        "granularity of cache save for incremental conversion",
+        "if you select None and another, then None is ignored.",
+        "Valid values: ${COMPLETION-CANDIDATES}"
+    }, order = Config.MIDDLE)
+    protected CacheLevel[] cacheLevel = {};
```

Comment:
Would it be `Set<CacheLevel>`? Then the following conversions of `Arrays.asList(cacheLevel)` would not be necessary.